setup.py

```diff
@@ -238,8 +238,12 @@ def setup_package():
         FULLVERSION, GIT_REVISION = get_version_info()
         metadata['version'] = FULLVERSION
     else:
-        if len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel':
-            # bdist_wheel needs setuptools
+        if (len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel' or
+                sys.version_info[0] < 3 and sys.platform == "win32"):
```

Comment:
Maybe a misunderstanding. Are we talking about the Windows SDKs [7.0](http://www.microsoft.com/en-us/download/details.aspx?id=3138) (for Python 2.6, 2.7, and 3.2) and [7.1](http://www.microsoft.com/en-us/download/details.aspx?id=8279) (for Python 3.3 and 3.4) or the [Visual C++ Compiler for Python 2.7](http://www.microsoft.com/en-us/download/details.aspx?id=44266)? AFAIK the former require `DISTUTILS_USE_SDK` but not setuptools (https://docs.python.org/2/distutils/apiref.html#module-distutils.msvccompiler), the latter requires setuptools >= 6 but no environment variables (https://pypi.python.org/pypi/setuptools/6.1#id4).